internal/render/render.go

```diff
@@ -18,16 +19,44 @@ var (
 )
 
 func init() {
+	initEmptyTmpls()
+	for _, name := range bindata.AssetNames() {
+		tmpls = template.Must(tmpls.Parse(string(bindata.MustAsset(name))))
+	}
+}
+
+// LoadCustomTemplate allows to load in custom templates from a specified path.
+func LoadCustomTemplate(path string) error {
+	path = strings.Trim(path, "/")
+	initEmptyTmpls()
```

Comment:
Do you need to call this again here? I think you can just overwrite `tmpls` when you call `Parse()` on with new file names. Ideally you shouldn't need to call `initEmptyTmpls` outside `init`.

If you only need to call it once, please revert `init` back its previous state.

Reply:
I have undone it, resulting in that you can re-use standard elements if you wanted to.

Reply:
Okay forget the above, I tested this but when you have a badly formatted template you try to overwrite it with the templates package panics instead of returning a proper error.